packages/ui/src/composables/useNumericProp.ts

```diff
@@ -0,0 +1,19 @@
+import { computed, getCurrentInstance } from 'vue'
+
+/**
+ * @param key key of props
+ * @returns Numeric type of key attribute
+ */
+export const useNumericProp = (key: string) => {
+  const numeric = getCurrentInstance()!.props?.[key] as string | number | undefined
+
+  const numericComputed = computed(() => {
+    if (numeric === undefined) {
```

Comment:
```suggestion
  const props = getCurrentInstance()!.props

  const numericComputed = computed(() => {
    const numeric = props?.[key] as string | number | undefined
  
    if (numeric === undefined) {
```

Here we're losing reactivity. Because prop is not a Ref or ComputedRef, but props are reactive, vue tracks reactivity when you access props[key]. Because reactivity tracked outsize of numericComputed computed, it will not be tracked and numericComputed will not update its value when prop is changed.